api/types/Cargo.toml

```diff
@@ -16,11 +16,11 @@ bcs = "0.1.3"
 hex = "0.4.3"
 indoc = "1.0.6"
 mime = "0.3.16"
+paste = "1.0.7"
```

Comment:
is this used somewhere?

Reply:
My bad, left from previous iteration of this PR. I'll audit all the deps.